packages/browser/src/login/oidc/redirectHandler/ErrorOidcHandler.ts

```diff
@@ -29,7 +29,7 @@ import {
   IRedirectHandler,
   ISessionInfo,
 } from "@inrupt/solid-client-authn-core";
-import { EventEmitter } from "stream";
+import { EventEmitter } from "events";
```

Comment:
```suggestion
import type { EventEmitter } from "events";
```
Since it's only used as a type, it should be marked as such.

Reply:
Good point. Fixed in e037b939cd02d5f4afa0c17d6ae2a593a90403d8